Source/Core/VideoCommon/VertexManagerBase.cpp

```diff
@@ -252,6 +252,17 @@ static void SetSamplerState(u32 index, bool custom_tex)
     state.anisotropic_filtering = 0;
   }
 
+  if (has_arbitrary_mips && SamplerCommon::AreBpTexMode0MipmapsEnabled(tm0))
+  {
+    // Apply a secondary bias calculated from the IR scale to pull inwards mipmaps
+    // that have arbitrary contents, eg. are used for fog effects where the
+    // distance they kick in at is important to preserve at any resolution.
+    state.lod_bias = state.lod_bias + std::log2((float)g_ActiveConfig.iEFBScale) * 256.f;
```

Comment:
`static_cast` should be used in new code added to the codebase

Reply:
Ok, honestly I just tried to copy the line just above that does `s32`. I know the struggle of wanting to change direction without restyling literally the whole project... new guys will always just copy what they see.